Misc/NEWS.d/next/Library/2018-12-09-17-04-15.bpo-17185.SfSCJF.rst

```diff
@@ -0,0 +1,2 @@
+Set ``__signature__`` on mock for `inspect` to get signature. Fix signature
```

Comment:
Can you use rst markup to reference inspect module?

I think it is :module: just before the backtick.

Reply:
Add :mod: that references inspect and verified it locally. Thanks 👍